### PR TITLE
Fix bug in listing policy rules

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

Slack request

## Pull Request Description

Remove `values()` on rules iteration as it's a list not a dictionary

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Not tested only manual inspection of the expected results from the API

### CODE REVIEW

**Review by emitreva-cb — APPROVED** [review]

File: examples/platform/policy_service_crud_operations.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def list_policies(cb, parser, args):
     for p in cb.select(Policy):
         print(u"Policy id {0}: {1} {2}".format(p.id, p.name, "({0})".format(p.description) if p.description else ""))
         print("Rules:")
-        for r in p.rules.values():
+        for r in p.rules:
             print("  {0}: {1} when {2} {3} is {4}".format(r.get('id'), r.get("action"),
```
Comment (emitreva-cb) [inline examples/platform/policy_service_crud_operations.py:75]: Could you please unify the type of quotes used - ""?

Comment (avanbrunt-cb) [reply examples/platform/policy_service_crud_operations.py]: I find it hilarious you noticed that and it was also single quotes in the entire file. Good catch!

File: examples/platform/policy_service_crud_operations.py [hunk 2]
```diff
@@ -71,8 +71,8 @@ def list_policies(cb, parser, args):
     for p in cb.select(Policy):
         print(u"Policy id {0}: {1} {2}".format(p.id, p.name, "({0})".format(p.description) if p.description else ""))
         print("Rules:")
-        for r in p.rules.values():
-            print("  {0}: {1} when {2} {3} is {4}".format(r.get('id'), r.get("action"),
+        for r in p.rules:
+            print("  {0}: {1} when {2} {3} is {4}".format(r.get("id"), r.get("action"),
```
Comment (kebringer-cb) [inline examples/platform/policy_service_crud_operations.py:75]: Verified fix works.
Could these two lines please be added to src/tests/uat/policy_uat.py at line 31?  Then list will be covered at least with a manual test. 

    # 0. List the policies
    $ python3 examples/platform/policy_service_crud_operations.py --profile PROFILE_NAME --verbose list

**Review by kebringer-cb — APPROVED** [review]
Don't need to re-review.

File: examples/platform/policy_service_crud_operations.py (hunk 2 above)
Comment (avanbrunt-cb) [reply examples/platform/policy_service_crud_operations.py:75]: done

**Review by abowersox-cb — APPROVED** [review]

**Pull request merged**